app/src/main/java/com/hedvig/app/ui/compose/theme/Colors.kt

```diff
@@ -13,4 +13,7 @@ val lavender400 = Color(0xffBE9BF3)
 val errorLight = Color(0xffDD2727)
 val errorDark = Color(0xffE24646)
 
+val textColorPrimary = Color(0xAB121212)
```

Comment:
We should probably use https://material-components.github.io/material-components-android-compose-theme-adapter/, in order to avoid having multiple sources of truth for what our theme is 😅

Reply:
Yes we are using the adapter now! These colors are unused, I'll remove them.